src/Riok.Mapperly/SymbolExtensions.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis;
+
+public static class SymbolExtensions
```

Comment:
There is a `SymbolExtensions` class in the `Helpers` directory, I think this could be included there.